orangecontrib/text/corpus.py

```diff
@@ -88,6 +90,14 @@ def _infer_text_features(self):
         self.set_text_features(include_feats)
 
     def extend_corpus(self, metadata, Y):
```

Comment:
Can `Y` be `None` by default? This is conceptually kind of like `Table.concatenate(..., axis=1)`?